src/objects/Zombie.js

```diff
@@ -1,8 +1,8 @@
 import Phaser from 'phaser'
 import HealthBar from './HealthBar';
 import EntityAnimations from './enums/EntityAnimations';
-import { collisionCategories, collisionMaskEverything } from './enums/Collisions';
-import Entity from "./Entity.js";
+import { collisionCategories } from './enums/Collisions';
+import Entity from './Entity.js';
```

Comment:
Is there a reason for single quotes over double quotes or just to keep consistency?
I use Phpstorm IDE which auto adds the classes and think it auto adds double quotes 🙄

Reply:
vs code is complaining to me about them... not sure php storm is the right IDE for js work, but it probs works just fine.
i'm adding an `.editorconfig` file in this PR, so hopefully we can get some consistency with double spaced indent...